modules/nodes-descheduler-installing.adoc

```diff
@@ -64,7 +64,21 @@ ifdef::nodes[]
 ====
 Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates`. Enabling both results in a conflict.
 ====
-... Optional: Expand the *Profile Customizations* section to set a custom pod lifetime value for the `LifecycleAndUtilization` profile. Valid units are `s`, `m`, or `h`. The default pod lifetime is 24 hours.
+... Optional: Expand the *Profile Customizations* section to set optional configurations for the descheduler.
+**** Set a custom pod lifetime value for the `LifecycleAndUtilization` profile. Use the *podLifetime* field to set a numerical value and a valid unit (`s`, `m`, or `h`). The default pod lifetime is 24 hours.
+
+**** Set a custom priority threshold to consider pods for eviction only if their priority is lower than a specified priority level. Use the *thresholdPriority* field to set a numerical priority threshold or use the *thresholdPriorityClassName* field to specify a certain priority class name.
++
+// TODO: Are there defaults for either of the fields that we should call out here?
++
+[NOTE]
+====
+You cannot specify both *thresholdPriority* and *thresholdPriorityClassName* for the descheduler.
+// TODO: Ask Jan - technically you CAN set both. What happens if you do (and what should we say here).
+====
++
+// There is also a 'devLowNodeUtilizationThresholds' - Ask Jan about it
```

Comment:
@ingvagabund Per the output in the previous comment, there is also a `devLowNodeUtilizationThresholds` field under `profileCustomizations`. And `namespaces`. Are either of those new and/or things that are supported/need called out in docs?